### ISSUE TITLE
Hide more settings behind Advanced button 

### ISSUE THREAD

**Issue by Safihre** [sabnzbd/sabnzbd#1082]
Recently I met some random SABnzbd users (at a job interview actually) who, even though being software developers themselves, found SAB not very easy.
I always thought that SAB was already quite easy, but maybe I was wrong. 
In particular the Config pages are quite overwhelming. Would it help to hide a lot of settings in the Config screens behind `Advanced` buttons? Just like we already do for Server options? 
Instead of moving things to the Specials page, we just hide them. Less options, less problems for not such experienced users. 

I imagine it similar to Sonarr's settings: if you switch on/off Advanced on 1 page it stays active throughout whole Config.

@sanderjo @shypike (and everyone else)

**Comment by sanderjo** [comment]
> Recently I met some random SABnzbd users (at a job interview actually)

Ah, cool: your interviewers were SAB users? 

> I always thought that SAB was already quite easy, but maybe I was wrong.

I think SAB is certainly easy as long as you don't use the (config) settings. And that is what I do: I don't change the Config settings, except for the RSS feed (for testing) and the newsserver settings.

So for me, the Advanced button could hide everything, even the SABnzbd Host (which I would replace which a radio button Localhost-only versus Remote-too.) because it's already in the Wizard.
So that is not a useful answer.

Other option: high-light / colorize settings that have been changed. And show the number of changed settings

**Comment by labrys** [comment]
Personally I like having the settings available, but I understand some people don't.  I don't see an issue with hiding features behind an advanced setting (I plan on doing similar for Medusa at some point).  In fact I don't see an issue with having several levels (e.g. basic, intermediate, advanced) to expose different levels of settings.

**Comment by Hellowlol** [comment]
I guess they havnt used nzbget. :D +1 for advance

**Comment by thezoggy** [comment]
i personally would rather see a 'reset to defaults' on each page (that way people can try stuff out and revert easily) instead of trying to dumb it down to make it more appealing to the masses.

**Comment by Safihre** [comment]
> We provide very little help in the app, most of it punts the user to a wiki doc... so we prob could be better about providing tooltips/help alerts..

Yes this is true and something we should work on.
What I do notice from the forum questions over the years that some people are changing options that they really shouldn't. Just because they saw them.

**Comment by labrys** [comment]
@thezoggy It's funny that you say that about adding more help in the app, I was thinking the same thing, particularly for when new settings are added, but I figured it was a big undertaking to add.

Even with added help/tooltips, I would still consider adding settings levels.  Things that are mandatory to configure I would make basic settings, most other settings I would make intermediate, and settings that shouldn't be changed unless a user understands the consequences or are too complicated to explain in a simple tooltip, I would put in advanced.

Between the levels and the help/tooltips I would definitely prioritize the help though.

**Comment by Safihre** [comment]
I created a branch where I implemented this idea `feature/easier`.
If anyone has time, would like some feedback!

**Comment by thezoggy** [comment]
I wanted to add more verbiage and change existing things awhile ago but due to everything needing translations... Shy told me to hold off until the next major bump.. by that time I didn't have the time. But now with all the active dev maybe it's time we finally over haul our settings. From the talks it  sounds like you want to mimic how sonarr does their stuff.. which works well and I'm fine with (we should use their v3 UI / phantom ui, as it is closer to Sab styling)

**Comment by Safihre** [comment]
Overhaul, in the sense of moving settings pages around? Or/and redesign?

**Comment by thezoggy** [comment]
Shuffle things around.. redo text strings.. etc. I did the 'redesign' with the global config (when we took the config away from the skins)

**Comment by sanderjo** [comment]
> I created a branch where I implemented this idea feature/easier.
> If anyone has time, would like some feedback!

I'm not the right person, because to me everything is Advanced, so probably better to ignore me:

- Advanced button: not really clear to me from the button when Advanced is on/off
- Not clear which options are advanced; I have to deduct. Why not Advanced options with another background color?
- With Advanced turned off, still a lot of options ... but ... probably better to ignore me
